contracts/utils/Counters.sol

```diff
@@ -37,6 +37,44 @@ library Counters {
         }
     }
 
+    /**
+     * @dev Returns the current value and increments the value in the storage.
+     */
+    function getAndIncrement(Counter storage counter) internal returns (uint256 counter_) {
```

Comment:
Can we use the variable name `value` for the return value?

Reply:
done